arbiter/util/util.cpp

```diff
@@ -140,6 +140,29 @@ std::unique_ptr<std::string> env(const std::string& var)
     return result;
 }
 
+bool parseBoolFromEnv(const std::string& var, bool defaultValue)
+{
+    auto value = env(var);
+    if (!value)
+    {
+        // env var is not set
+        return defaultValue;
+    }
+    if (value->empty())
+    {
+        // env var is set to the empty string; interpret as false
+        return false;
```

Comment:
I wonder if this is backwards from what would be expected, e.g. if I run `VERBOSE ./my-app` I might expect that I've set `VERBOSE` to be *enabled* rather than disabled.